test/parallel/test-fs-stat-bigint.js

```diff
@@ -60,15 +60,17 @@ function verifyStats(bigintStats, numStats) {
         numStats.isSymbolicLink()
       );
     } else if (key.endsWith('Ms')) {
+      console.log(bigintStats);
+      console.log(numStats);
```

Comment:
Would prefer not to add the console.log statements here

Reply:
done!